src/quo/components/dropdowns/dropdown_input/view.cljs

```diff
@@ -0,0 +1,68 @@
+(ns quo.components.dropdowns.dropdown-input.view
+  (:require
+    [quo.components.dropdowns.dropdown-input.properties :as properties]
+    [quo.components.dropdowns.dropdown-input.style :as style]
+    [quo.components.icon :as icon]
+    [quo.components.markdown.text :as text]
+    [quo.theme :as theme]
+    [react-native.core :as rn]))
+
+(defn- view-internal
+  [{:keys [state theme on-press icon?
+           label? blur?
+           accessibility-label]
+    :or   {state  :default
+           icon?  true
+           label? true
+           blur?  false}
+    :as   props}
+   label header-label]
```

Comment:
Any particular reason label and header label aren't passed in the map?

Reply:
to be honest i thought we have a convention to pass such info as labels outside of maps to imitate api of core components like rn/text, where you also pass label outside of map

Reply:
yeah that's true, I think I had not seen any component with 2 args like this, i.e `[map arg arg]` but most components like button, dropdown etc take the form of `[map arg]`. 
I don't. think it matters so much, I wonder for consistency is it better to put header-label in the map instead? how does the other dropdown handle it, let's match that if it has a similar header-label?